src/python/pants/goal/run_tracker.py

```diff
@@ -101,10 +103,60 @@ def start(self, run_start_time: float, specs: List[str]) -> None:
                 "path": get_buildroot(),
                 "version": VERSION,
                 "cmd_line": cmd_line,
+                "goals": self.goals,
```

Comment:
@asherf is this addition still problematic wrt toolchain code?

Reply:
Yes
There is no reason to change/migrate this field right now.

Reply:
@benjyw FYI.
At any case this change teems unrelated to the topic of this PR (and breaks stuff on the toolchain side for reason we discussed internally)...
I suggest we revert this specific change.

Reply:
Ooops yes, that was cruft. Good catch.